index.js

```diff
@@ -92,6 +93,16 @@ module.exports = {
     // rely on implicit returns when return booleans or undefined.
     'consistent-return': ['error', { treatUndefinedAsUnspecified: true }],
     'eol-last': ['error', 'always'],
+
+    'import/order': [
+      'warn',
+      {
+        alphabetize: { order: 'asc', caseInsensitive: true },
+        'newlines-between': 'always',
+        pathGroups: [{ pattern: '{,~}/**', group: 'internal' }],
```

Comment:
What kind of imports does the comma support capturing?

Reply:
This is bash expansion - this doesn't match a literal comma, but matches both `/**` and `~/**`.